14/umbraco-cms/customizing/section-trees/sections/section-view.md

```diff
@@ -12,17 +12,103 @@ This page is a work in progress and may undergo further revisions, updates, or a
 
 <figure><img src="../../../.gitbook/assets/section-views.svg" alt=""><figcaption><p>Section View</p></figcaption></figure>
 
-**Manifest**
+## Creating a custom Section View
+
+### Manifest
+
```

Comment:
What do you think about using the [Tabs option in GitBook](https://docs.gitbook.com/content-editor/blocks/tabs) for presenting the two ways of creating the Manifest file?

Typescript in one tab, and JSON in the other 🤔 
Just an idea! 😁

Reply:
Oh yep, I like the idea! I updated it.